os/board/rtl8721csm/src/component/common/api/wifi_interactive_mode.c

```diff
@@ -415,6 +423,20 @@ int8_t cmd_wifi_ap(trwifi_softap_config_s *softap_config)
 	return ret;
 }
 
+void wlan_store_connect_info(void)
+{
+	u8 *ifname[1] = {(u8*)WLAN0_NAME};
+	rtw_wifi_setting_t setting;

```

Comment:
I think it is better to initialize the `setting` for checking `setting.mode` properly.

Reply:
The `setting` is memset to '0' and The condition of  `setting.mode` is placed after `wifi_get_setting()`.